usefulUtilities/hiFiCalendar/README.md

```diff
@@ -30,5 +30,28 @@ This app lets you integrate with the Google Calendar API to query meeting room s
 - Initial Release
 
 ## TO DO
-1. Remove Room Label Entity and just have a schedule entity that changes colors in next version
-2. Refactor authorization to allow app to run in the background.
\ No newline at end of file
+1. Simplify setup experience.
+
+## Technical Details:
+Here's what I'd expect here:
```

Comment:
is all of this accurate? if it is, you don't need this line. if it isn't, please make it so :)